cmd/calyptia/update.go

```diff
@@ -5,17 +5,18 @@ import "github.com/spf13/cobra"
 func newCmdUpdate(config *config) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "update",
-		Short: "Update aggregators, pipelines, etc.",
+		Short: "Update aggregators, pipelines, core instances etc.",
 	}
 
 	cmd.AddCommand(
 		newCmdUpdateProject(config),
 		newCmdUpdateAgent(config),
-		newCmdUpdateAggregator(config),
+		//newCmdUpdateAggregator(config),
```

Comment:
don't comment this out for now.

Reply:
this guy actually is core_instance command that we were using "aggregator" before, i transfered the logic to the new comand